variables.tf

```diff
@@ -80,4 +80,9 @@ variable "progress_deadline_seconds" {
 variable "service_port" {
   type    = number
   default = 32543
+}
+
+variable "redis_connection_string" {
+  type        = string
+  description = "The redis connection string."
```

Comment:
yes good point, the default for this would be `redis://127.0.0.1:6379/filemetadata`. Will update

Reply:
This is actually sort of tricky because we don't set GORILLA_CACHE by default ( it's a `noop://` ) , but having this default here _would_ set it to use the internal redis cache, which I'm not sure we want https://github.com/wandb/core/blob/464487e2e3166a154aac21ad6707f871c501b734/services/local/settings.go#L680

Reply:
actually from testing looks like using an empty string default here works fine

Reply:
Yep, let's not enable it when redis is at 127.0.0.1:6379

Reply:
Ah, yep you're right.  This is the proper default as it gets turned into the `REDIS` env var.